### ISSUE TITLE
Ikea Gunnarp not showing in devices (domoticz)

### ISSUE THREAD

**Issue by Ruftie** [Koenkk/zigbee-herdsman-converters#1353]
In Domoticz the part of  **extend: generic.light_onoff_brightness_colortemp,** part isn't showing in domoticz.
It is paired in Zigbee2Mqtt butt the isnt showed in devices part of domoticz. 

I can put it on or off bij JSON commands and removed and rejoined the device multiple times.

**Comment by Koenkk** [comment]
`extend` is converted to `toZigbee`, `supports` and `fromZigbee` so it is never exposed (https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices.js#L11172)

**Comment by Ruftie** [comment]
I will be a Noob for asking, but I don't understand your answer, sorry.

**Comment by Ruftie** [comment]
never mind fixt it :) the issues was in the plug-in to domoticz from https://github.com/stas-demydiuk/domoticz-zigbee2mqtt-plugin/
